search/lib/api/index.js

```diff
@@ -32,6 +32,14 @@ routes.use(provider.routes);
 routes.use(stac.routes);
 routes.use(wfs.routes);
 
+const cloudroutes = express.Router();
+cloudroutes.use('/docs', express.static(path.join(__dirname, '../../docs/index.html')));
```

Comment:
I just checked and http://cmr.earthdata.nasa.gov/stac/docs doesn't even work, although it works locally. So it won't work here.   Should create issue for this

Reply:
Filed CMR-7150.  It doesn't work in SIT(403) and UAT(502) either.